src/ledger/LedgerTxn.cpp

```diff
@@ -2394,6 +2415,8 @@ LedgerTxnRoot::Impl::tableFromLedgerEntryType(LedgerEntryType let)
         return "trustlines";
     case CLAIMABLE_BALANCE:
         return "claimablebalance";
+    case LIQUIDITY_POOL:
```

Comment:
Missed the related change on line 2482. Maybe you can update that line to use `xdr_traits<LedgerEntryType>` to avoid this in the future.

Reply:
Done